src/ia-account-settings.ts

```diff
@@ -0,0 +1,804 @@
+            class="form-control"
+            id="screenname"
+            name="screenname"
+            .value="${this.userData.screenname}"
+            @input=${this.setScreenname}
+            @blur=${this.validateScreenname}
+          />
+          <span class="error-field">${this.fieldsError.screenname}</span>
+        </div>
+
+        <div class="form-element">
+          <label for="email">
+            Change email <small>(verification will be required)</small>
+          </label>
+          <input
+            type="email"
+            class="form-control"
+            id="email"
+            name="email"
+            .value="${this.userData.email}"
+            @input=${this.setEmail}
+            @blur=${this.validateEmail}
+          />
+          <span class="error-field">${this.fieldsError.email}</span>
+        </div>
+
+        <div class="form-element">
+          <label for="password">
+            Change Internet Archive / Open Library password
+          </label>
+          <input
+            type="password"
+            class="form-control"
+            id="password"
+            name="password"
+            autocomplete="new-password"
+            @input=${this.setPassword}
+            @blur=${this.validatePassword}
+          />
+          <img
+            class="password-icon"
+            src="https://archive.org/images/eye-crossed.svg"
+            @click=${(e: Event) =>
+              togglePassword(e, this.passwordField as HTMLInputElement)}
+            alt="View text"
+          />
+          <span class="error-field">${this.fieldsError.password}</span>
+        </div>
+
+        <div class="form-element">
+          <label>Set borrow history</label>
+          <input
+            type="checkbox"
+            id="borrow-history"
+            name="borrow-history ${this.loanHistoryFlag}"
+            .checked=${this.loanHistoryFlag === 'public' ||
+            this.loanHistoryFlag === true}
+            @click=${this.setBorrowHistory}
+          />
+          <label for="borrow-history"> Visible to the public</label>
+        </div>
+
+        <div class="form-element newsletter">
+          <label>Newsletter subscriptions</label>
+          <p>
+            Stay up to date with what's happening at the Internet Archive by
+            signing up for our free newsletters.
+          </p>
+          ${this.mailingListsTemplate}
+        </div>
+
+        <div class="form-element">
+          <label for="linked-account"
+            >Linked 3rd party accounts (e.g. Google)</label
+          >
+          ${this.linkedProviders.length
+            ? this.linkedAccountTemplate
+            : 'You have no linked accounts'}
+        </div>
+
+        <div
+          class="form-element delete-link ${this.attemptToDelete ? 'hide' : ''}"
+        >
+          <a
+            href="javascript:void(0)"
+            @click=${() => {
+              this.attemptToDelete = true;
+              window.scrollTo();
```

Comment:
Q: What does `window.scrollTo()` without args do? I thought it required coordinates to be provided.